scipy/sparse/linalg/_expm_multiply.py

```diff
@@ -136,13 +192,14 @@ def expm_multiply(A, B, start=None, stop=None, num=None, endpoint=None):
     array([ 2.71828183,  1.        ])
     """
     if all(arg is None for arg in (start, stop, num, endpoint)):
-        X = _expm_multiply_simple(A, B)
+        X = _expm_multiply_simple(A, B, traceA=traceA)
     else:
-        X, status = _expm_multiply_interval(A, B, start, stop, num, endpoint)
+        X, status = _expm_multiply_interval(A, B, start, stop, num,
+                                            endpoint, traceA=traceA)
     return X
 
 
-def _expm_multiply_simple(A, B, t=1.0, balance=False):
+def _expm_multiply_simple(A, B, t=1.0, traceA=None, balance=False):
```

Comment:
I might have opted to put the new argument at the end of the list in case of positional incantations, but the function is private I suppose and `git grep` doesn't show any places where the positional usage gets that far in the arg list.

Reply:
I don't really care. The reason I put it before `balance` is, that `balance` is not implemented anyway.